frontend/src/environments/environment.ts

```diff
@@ -14,7 +20,13 @@ export const environment: DataDiscoveryEnvironment = {
     secondLogoUrl: '',
     links: [{ label: 'agrobrc', url: 'https://www.agrobrc-rare.org/' }]
   },
-  resourceModule: RareModule,
+  documentComponent: GenericDocumentComponent,
```

Comment:
We used to have the same config as RARe. Any reason why you switched to GenericDocumentComponent and to GenericBasketComponent? Is it to make testing easier because the generic components have fewer dependencies?

Reply:
Yes exactly